pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -58,15 +58,14 @@ func checkHyperVInstalled() (bool, error) {
 		return false, errors.New("Hyper-V not installed")
 	}
 
-	// Check if Hyper-V's Virtual Machine Management Service is running
-	checkVmmsRunning := `@(Get-Service vmms).Status`
-	stdOut, _, err = powershell.Execute(checkVmmsRunning)
+	checkVmmsExists := `@(Get-Service vmms).Status`
+	stdOut, _, err = powershell.Execute(checkVmmsExists)
 	if err != nil {
 		logging.Debug(err.Error())
-		return false, errors.New("Failed checking if Hyper-V is running")
+		return false, errors.New("Failed checking if Hyper-V management service exists")
 	}
-	if strings.TrimSpace(stdOut) != "Running" {
-		return false, errors.New("Hyper-V Virtual Machine Management service not running")
+	if strings.Contains(stdOut, "Get-Service") {
```

Comment:
The error message goes to stderr, not stdout, I've tested with this change and it worked:
```
diff --git a/pkg/crc/preflight/preflight_checks_windows.go b/pkg/crc/preflight/preflight_checks_windows.go
index ebf8cecd..50b3b236 100644
--- a/pkg/crc/preflight/preflight_checks_windows.go
+++ b/pkg/crc/preflight/preflight_checks_windows.go
@@ -59,12 +59,12 @@ func checkHyperVInstalled() (bool, error) {
        }
 
        checkVmmsExists := `@(Get-Service vmms).Status`
-       stdOut, _, err = powershell.Execute(checkVmmsExists)
+       _, stdErr, err := powershell.Execute(checkVmmsExists)
        if err != nil {
                logging.Debug(err.Error())
                return false, errors.New("Failed checking if Hyper-V management service exists")
        }
-       if strings.Contains(stdOut, "Get-Service") {
+       if strings.Contains(stdErr, "Get-Service") {
                return false, errors.New("Hyper-V management service not available")
        }
```

Reply:
bahaviour as before... approved?

Reply:
Ah, got it... changed from `stdOut` to `stdErr`.